tsconfig.json

```diff
@@ -1,16 +1,11 @@
 {
   "compilerOptions": {
-    "allowSyntheticDefaultImports": true,
     "target": "esnext",
     "isolatedModules": true,
     "moduleResolution": "node16",
     "allowImportingTsExtensions": true,
     "noEmit": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
```

Comment:
those are linting concerns and we should use a linter for those kind of things, I think it's best to remove them for now from here and consider adding ESlint to the project at some point